Intro-Full/Exercises/kokkoskernels/SpILUK/Begin/CMakeLists.txt

```diff
@@ -5,6 +5,12 @@ find_package(KokkosKernels REQUIRED)
 
 include(../../common.cmake)
 
+if(NOT DEFINED KokkosKernels_SOURCE_DIR)
```

Comment:
@vqd8a did you define KokkosKernels_SOURCE_DIR in local scripts? I tried building this locally without doing so and ran into this error:

```
CMake Error at CMakeLists.txt:9 (message):
  Please specify KokkosKernels_SOURCE_DIR to include necessary test headers.
```

Reply:
@ndellingwood I just wanted to include the header files in ```test_common``` but I don't know a good way to do it. 
So I just manually define ```-DKokkosKernels_SOURCE_DIR=~/Kokkos/kokkos-kernels``` with the ```cmake``` command .

Reply:
@ndellingwood Is there another way to do this?

Reply:
@ndellingwood Should I modify the ```run_installlibs_cmake.sh``` to define ```KokkosKernels_SOURCE_DIR```?

Reply:
@vqd8a I think that'll work for these exercises, use the env variable `KOKKOSKERNELS_PATH` i.e.
`-DKokkosKernels_SOURCE_DIR=$KOKKOSKERNELS_PATH`

Reply:
@vqd8a I just tried compiling with that change and ran into this error:
```
/Users/ndellin/Research/kokkos_items/kokkos-tutorials/Intro-Full/Exercises/kokkoskernels/SpILUK/Begin/sparse_iluk.cpp:125:29: error: 'KokkosSparse' does not name a type; did you mean 'KokkosKernels'?
     using crsmat_t        = KokkosSparse::CrsMatrix<scalar_t, lno_t, Kokkos::DefaultExecutionSpace, void, size_type>;
                             ^~~~~~~~~~~~
                             KokkosKernels
...
```

Reply:
@ndellingwood I added `-DKokkosKernels_SOURCE_DIR="${KOKKOSKERNELS_PATH}"` to the ```run_installlibs_cmake.sh``` and ran this script in the ```Solution``` directory. It was built without error. I believe you saw the error because you ran it in the ```Begin``` directory which contains incomplete code.

Reply:
@vqd8a maybe the easiest thing to do is to copy/paste this code from the spgemm exercise?
https://github.com/kokkos/kokkos-tutorials/blob/8cc7985f42084a42b470d331cd0b47f98eeefb7f/Intro-Full/Exercises/kokkoskernels/SpGEMM/Begin/spgemm.cpp#L69-L168

Reply:
That could remove the need to include the test_common directory

Reply:
> @ndellingwood I added `-DKokkosKernels_SOURCE_DIR="${KOKKOSKERNELS_PATH}"` to the `run_installlibs_cmake.sh` and ran this script in the `Solution` directory. It was built without error. I believe you saw the error because you ran it in the `Begin` directory which contains incomplete code.

@vqd8a my mistake, didn't realize it was incomplete. Let me try locally real quick

Reply:
@ndellingwood Thanks. I think it should work now.

Reply:
That worked, thanks Vinh! I'll merge now.

Reply:
Thanks, Nathan.